code/crates/core-consensus/src/types.rs

```diff
@@ -133,3 +133,137 @@ pub enum LivenessMsg<Ctx: Context> {
     PolkaCertificate(PolkaCertificate<Ctx>),
     SkipRoundCertificate(RoundCertificate<Ctx>),
 }
+
+#[cfg(feature = "borsh")]
+mod _borsh {
```

Comment:
Let's move this module into its own file at `code/crates/core-consensus/src/ser/borsh.rs`